raster-foundry/raster-foundry
# Lambda Publishing Fixes, Upgrade Everything to Scala 2.12

## Overview

Some minor fixes and adding logging to enable publishing the lambda function to AWS and upgrading all subprojects to scala 2.12

There are a few fixes for compiler warnings, but most of those are deferred to a later task.

### Checklist


## Testing Instructions

- Make sure everything compiles
- Test out functionality for the application (view projects, etc.)

Closes #4602
